src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -660,6 +661,10 @@ private void commonColumnsVerification(String headerName, String columnValue, St
                 assertTrue(StringUtils.isBlank(columnValue));
                 break;
             }
+            case "substudyMemberships": {
+                assertTrue(StringUtils.isBlank(columnValue));
+                break;
+            }
```

Comment:
Should we have a test where we assign a substudy and external ID to make sure this is being read, serialized, and exported properly?

Reply:
I didn't see this being done for data groups, so was not sure if we could manipulate users in the study. I can give it a try.

Reply:
Added!